jest.config.js

```diff
@@ -0,0 +1,189 @@
+// For a detailed explanation regarding each configuration property, visit:
+// https://jestjs.io/docs/en/configuration.html
+
+module.exports = {
+  // All imported modules in your tests should be mocked automatically
+  // automock: false,
+
+  // Stop running tests after `n` failures
+  // bail: 0,
+
+  // The directory where Jest should store its cached dependency information
+  // cacheDirectory: "/private/var/folders/tx/xm0ncpnx3cddx9blptw3w8x80000gn/T/jest_dx",
+
+  // Automatically clear mock calls and instances between every test
+  clearMocks: true,
+
+  // Indicates whether the coverage information should be collected while executing the test
+  // collectCoverage: false,
+
+  // An array of glob patterns indicating a set of files for which coverage information should be collected
+  // collectCoverageFrom: undefined,
+
+  // The directory where Jest should output its coverage files
+  coverageDirectory: "coverage",
+
+  // An array of regexp pattern strings used to skip coverage collection
+  coveragePathIgnorePatterns: [
+    "/node_modules/",
+    "/dist/",
+    "/tests/",
+    "/scripts/",
+    "__tests__"
+  ],
```

Comment:
coverage should exclude test files as well? `*.test.js` ?

Reply:
no..this is for the folder which will has other js files as well, so don't want that all files in folder.
the key which controls the test file names is **testMatch**